docs/releases.md

```diff
@@ -0,0 +1,64 @@
+# Releases
+This document covers making releases for distribution.
+
+## Windows Warning
+The process for creating releases is highly automated and can be run on
+any platform. However, while the releases built on macOS/Linux for Windows work
+as expected, the releases built on Windows for macOS/Linux require changing
+file permissions to run. This is because Linux file permissions don't exist
+on Windows and default to not being executable on macOS and Linux. **Publishing
+macOS and Linux releases built on Windows are strongly discouraged.**
+If you are on Windows, a Linux virtual machine, such as [Windows Subsystem for Linux (WSL)](https://docs.microsoft.com/en-us/windows/wsl/about)
```

Comment:
after the first comma I'd add "you can use", or something similar

Reply:
I am not sure about this request. I interpret my wording as "these are examples of tools that work, but others exist" while I would interpret your requested change as "these are the tools that work and no others exist or are recommended".

Reply:
It does not have to be exactly "you can use", it's just that "a Linux virtual machine, such as WSL or Virtualbox." isn't a complete sentence (and also doesn't imply to me that there are any other options beside a VM). So my suggestion is to add something to make it one. It could say "it's possible to use a Linux virtual machine", or "one option is a Linux virtual machine", or anything else you like.

Reply:
Uh, oops. There should be more words after all that. More has been added.

Reply:
That's perfect thanks :+1: